.github/workflows/generate-summary.yaml

```diff
@@ -525,28 +543,25 @@ jobs:
           echo "label=$LABEL" >> $GITHUB_OUTPUT
 
       - name: Add Prefix Label to New Issue
-        uses: actions/github-script@v7
         if: ${{ inputs.prefix != '' }}
+        uses: ./.github/actions/common/save-label
         with:
-          script: |
-            github.rest.issues.addLabels({
-              issue_number: ${{ steps.create-issue.outputs.issue-number }},
-              owner: context.repo.owner,
-              repo: context.repo.repo,
-              labels: ['${{ steps.prefix-label.outputs.label }}']
-            })
+          label: "'${{ steps.prefix-label.outputs.label }}'"
+          output: ${{ steps.label-aggregation-preset.outputs.label_file }}
 
       - name: Add Staging Label to New Issue
-        uses: actions/github-script@v7
         if: ${{ github.workflow == 'Staging' }}
+        uses: ./.github/actions/common/save-label
         with:
-          script: |
-            github.rest.issues.addLabels({
-              issue_number: ${{ steps.create-issue.outputs.issue-number }},
-              owner: context.repo.owner,
-              repo: context.repo.repo,
-              labels: ['staging']
-            })
+          label: "'staging'"
+          output: ${{ steps.label-aggregation-preset.outputs.label_file }}
+
+      - name: Add Build warnings Label to New Issue
```

Comment:
nit: I think this would make sense to be added when we perform the build warning label check

Reply:
As in execute this step right after the `build warning label check` step?

Reply:
yeah I think that would make the most sense